src/h5web/providers/utils.ts

```diff
@@ -91,3 +92,19 @@ export function flattenValue(
   const dims = slicedDims || entity.shape;
   return (value as unknown[]).flat(dims.length - 1);
 }
+
+export function handleAxiosError<T>(
```

Comment:
Similar to `handleError` in `h5web/utils` but handles axios error responses with specific status codes instead of JS errors with specific messages.